README.md

```diff
@@ -85,25 +85,37 @@ This section describes how to run geth node with ewasm backend.
 The key component to add ewasm to geth is [EVMC](https://github.com/ethereum/evmc).
 Aleth supports EVMC out of the box, but geth not yet.
 
-1. Build geth with EVMC
+### Releases of geth with EVMC support
+
+The geth for ewasm testnet consist of following branches / PRs:
+
+1. Support for EVMC: https://github.com/ethereum/go-ethereum/pull/17954
+2. Take VM config from BlockChain: https://github.com/ethereum/go-ethereum/pull/17955
+3. params: Include EVMC version in the geth version: https://github.com/ethereum/go-ethereum/compare/master...ewasm:evmc-version
+4. tests: Add flag to use EVMC for state tests: https://github.com/ethereum/go-ethereum/compare/master...ewasm:evmc-tests-flag
+5. Allow setting EVMC by env var (TBD, workaround)
+6. acct-range-at (TBD)
+
+The pack of above changes are released regurally as git tag `ewasm-testnet-vX`:
+
+- TBD: [`ewasm-testnet-v1`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-v1)
+
+Deployed:
+
+- Production: TBD: [`ewasm-testnet-production`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-v1)
+- Staging: TBD: [`ewasm-testnet-staging`](https://github.com/ewasm/go-ethereum/tree/ewasm-testnet-v1)
+
+
+#### Building
+
+1. Build geth
 
-	Checkout the `evmc` branch of [this geth fork](https://github.com/chfast/go-ethereum/tree/evmc) ([PR](https://github.com/ethereum/go-ethereum/pull/17050)):
```

Comment:
Here we could specify again that we need to get the `ewasm-testnet-v1` branch just to be explicit in the building process.